rllib/algorithms/tests/test_algorithm.py

```diff
@@ -183,6 +182,43 @@ def _has_policies(w):
                 self.assertTrue(pol0.action_space.contains(a))
                 test.stop()
 
+                # After having added 2 policies, try to restore the Algorithm,
+                # but only with 1 of the originally added policies (plus the initial
+                # p0).
+                if i == 2:
+
+                    def new_mapping_fn(agent_id, episode, worker, **kwargs):
+                        return f"p{choice([0, 2])}"
+
+                    test2 = pg.PG.from_checkpoint(
+                        checkpoint=checkpoint,
+                        policies=["p0", "p2"],
+                        policy_mapping_fn=new_mapping_fn,
```

Comment:
Need to redefine these here as otherwise, RLlib has no means of knowing how to map with this reduced set of policies.